tests/generate_test.js

```diff
@@ -161,16 +190,15 @@ describe('generateCSS', () => {
             WebkitAlignItems: 'center',
             justifyContent: 'center',
         }], '.foo{' +
-            '-ms-flex-pack:center !important;' +
-            '-webkit-box-align:center !important;' +
-            '-ms-flex-align:center !important;' +
             '-webkit-box-pack:center !important;' +
-            'display:flex !important;' +
```

Comment:
Oh actually now that I look at this more closely, it seems like `display: flex` should go after `display: -webkit-box`? Maybe we should ensure that. I can do that separately from this change, though.

Other than that, all of these changes seem quite reasonable to me.